### ISSUE TITLE
Brief Sr. leaders and seek permission to start a pilot program.

### ISSUE THREAD

**Issue by Johnny2136** [CS-C-BDD-TDD/Behavior-Driven-Development#10]
The end of phase I We get to see if leaders will let us pilot a real application.

**Comment by drtran** [comment]
Yes. I'd like to discuss that with you today when we meet. I am allocating all day being at the Centra office. However, I'd like to beat the 3 PM on 66.
